### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by ryantuludjian** [ho-dev/HattrickOrganizer#851] [Q&A]
I'm new to github can anyone explain how to use the hattrick I installed HO but I don't know how to set it up

**Comment by akasolace** [comment]
hello,
You can read the [short documentation](https://ho.readthedocs.io/en/latest/)and/or ask for help on dedicated HT forum [HT forum](https://www83.hattrick.org/Forum/Read.aspx?t=17377058&v=4)
